### ISSUE TITLE
Detect and warn about lack of feature ids in local GeoJSON

### ISSUE THREAD

**Issue by nrotstan** [osmlab/maproulette3#547]
* When building a challenge from local GeoJSON, detect and warn about lack of feature ids as this will lead to task duplication on rebuild

* Do not allow rebuild of challenge from local GeoJSON lacking feature ids

**Comment by mvexel** [comment]
This still seems relevant @CollinBeczak @jschwarz2030 @AndrewPhilbin ? Perhaps a good first issue for a new contributor?

**Comment by CollinBeczak** [comment]
I don't think this would be a good first issue. Would either require some weird frontend validation, or backend validation that is connected to the frontend and throws an error modal that allows the user to either cancel the request or continue. I honestly think the fix would be more complex than it looks at face value.

**Comment by CollinBeczak** [comment]
A good solution (i think) would be to add a parameter to the query `checkFeatures: true`, it would return either an error or unique response that makes the modal show up with the "cancel" or "continue" options, and then continue will rerun the query with `checkFeatures: false`.
